src/main/java/com/juansecu/opentoonix/shared/utils/mail/EmailUtil.java

```diff
@@ -0,0 +1,29 @@
+package com.juansecu.opentoonix.shared.utils.mail;
+
+/* --- Third-party modules --- */
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmailUtil {
+
+    private final JavaMailSender mailSender;
+
+    @Value("${ACCOUNTS_FROM_EMAIL_ADDRESS}")
+    private String fromEmailAddress;
+
+    public EmailUtil(JavaMailSender mailSender) {
+        this.mailSender = mailSender;
+    }
+
+    public void sendEmail(String to, String subject, String text) {
+        SimpleMailMessage email = new SimpleMailMessage();
+        email.setFrom(fromEmailAddress);
```

Comment:
From email address should be passed as a parameter, instead of retrieving it directly in this class.